openreview/venue/venue.py

```diff
@@ -1144,6 +1144,9 @@ def ithenticate_create_and_upload_submission(self):
                 except Exception as err:
                     iThenticate_edge.label = "Created"
                     iThenticate_edge = self.client.post_edge(iThenticate_edge)
+
+            else:
+                print(f"Submission {submission.id} already has edges associated with it")
```

Comment:
can you print the label of the edge too?

Reply:
Done